pkg/scheduler/partition_test.go

```diff
@@ -1827,7 +1846,8 @@ func TestPreemption(t *testing.T) {
 	assert.NilError(t, err, "failed to add ask alloc-3 to app-2")
 
 	// delay so that preemption delay passes
-	time.Sleep(100 * time.Millisecond)
+	// also make the delay 1 second to have a minimum non-zero resource*seconds measurement for preempted resources
+	time.Sleep(1 * time.Second)
```

Comment:
nit: `time.Sleep(time.Second)`

Reply:
Thanks @pbacsko good catches! I just uploaded new rev to address both of them. Appreciate your help very much.